statsmodels/tsa/kalmanf/kalman_loglike.pyx

```diff
@@ -225,8 +225,8 @@ def kalman_filter_complex(dcomplex[:] y,
         dcomplex[::1,:] alpha = PyArray_ZEROS(2, mshape, cnp.NPY_CDOUBLE, FORTRAN)
         int lda = alpha.strides[1]/sizeof(dcomplex)
         # initial variance
-        dcomplex[::1,:] P = dot(pinv(identity(r**2)-kron(T_mat, T_mat)),
-                            dot(R_mat,R_mat.T).ravel('F')).reshape(r,r, order='F')
+        dcomplex[::1,:] P = asfortranarray(dot(pinv(identity(r**2)-kron(T_mat, T_mat)),
+                                           dot(R_mat,R_mat.T).ravel('F')).reshape(r,r, order='F'))
```

Comment:
does this need to go into 0.8?
@bashtage @ChadFulton 
I prefer to put the positive definite check into 0.9